### PR TITLE
fix a typo in a comment

### PR DESCRIPTION
**Pull request by adsick**
not sure whether it was a "must return "0"" or "returns "0"" so correct if wrong

### CODE REVIEW

File: near-contract-standards/src/fungible_token/core_impl.rs [hunk 1]
```diff
@@ -48,7 +48,7 @@ pub trait FungibleTokenContract {
     /// Returns the total supply of the token in a decimal string representation.
     fn ft_total_supply(&self) -> U128;
 
-    /// Returns the balance of the account. If the account doesn't exist must returns `"0"`.
+    /// Returns the balance of the account. If the account doesn't exist must return `"0"`.
```
Comment (willemneal) [inline near-contract-standards/src/fungible_token/core_impl.rs:51]:
```suggestion
    /// Returns the balance of the account. If the account doesn't exist `"0"` must be returned.
```

Comment (austinabell) [reply near-contract-standards/src/fungible_token/core_impl.rs:51]:
```suggestion
    /// Returns the balance of the account. If the account doesn't exist, `"0"` must be returned.
```
Comma seems important to be correct

**Comment by austinabell** [comment]
Thanks for noticing this! Will merge once the comment about wording is addressed :)

File: near-contract-standards/src/fungible_token/core_impl.rs [hunk 2]
```diff
@@ -48,7 +48,7 @@ pub trait FungibleTokenContract {
     /// Returns the total supply of the token in a decimal string representation.
     fn ft_total_supply(&self) -> U128;
 
-    /// Returns the balance of the account. If the account doesn't exist must returns `"0"`.
+    /// Returns the balance of the account. If the account doesn't exist `"0"` must be returned.
```
Comment (austinabell) [inline near-contract-standards/src/fungible_token/core_impl.rs:51]:
```suggestion
    /// Returns the balance of the account. If the account doesn't exist, `"0"` must be returned.
```
Comma is required to make sense, but I'm happy to fix in a follow up if you prefer

**Review by austinabell — APPROVED** [review]

**Pull request merged**